src/qt/guiutil.cpp

```diff
@@ -768,23 +768,66 @@ QString formatDurationStr(int secs)
     return strList.join(" ");
 }
 
+void sortLocaleAware(QStringList &sList)
+{
+    std::sort(sList.begin(), sList.end(), [](const QString &s1, const QString &s2){
+        return s1.localeAwareCompare(s2) < 0;
+    });
+}
+
 QString formatServicesStr(quint64 mask)
 {
     QStringList strList;
 
     for (const auto& flag : serviceFlagsToStr(mask)) {
+        if (flag == "NETWORK_LIMITED"){
+        strList.append(QString::fromStdString("LIMITED"));
+        }
+        else
+        {
         strList.append(QString::fromStdString(flag));
+        }
+        sortLocaleAware(strList);
     }
 
     if (strList.size())
-        return strList.join(" & ");
+        return strList.join(" ");
     else
         return QObject::tr("None");
 }
 
+QString shortFormatServicesStr(quint64 mask)
+{
+    QStringList strList;
+
+    for (const auto& flag : serviceFlagsToStr(mask)) {
+        if (flag == "NETWORK"){
+        strList.append(QString::fromStdString("N"));
+        }
+        if (flag == "NETWORK_LIMITED"){
+        strList.append(QString::fromStdString("L"));
+        }
+        if (flag == "BLOOM"){
+        strList.append(QString::fromStdString("B"));
+        }
+        if (flag == "WITNESS"){
+        strList.append(QString::fromStdString("W"));
+        }
+        if (flag == "UNKNOWN"){
+        strList.append(QString::fromStdString("U"));
+        }
+        sortLocaleAware(strList);
+    }
+
+    if (strList.size())
+        return strList.join("");
+    else
+        return QObject::tr(""); //silence is golden
```

Comment:
This conditional seems pointless

Reply:
@luke-jr ,

- Please actually download and run a PR before reviewing.
- Before arbitrarily declaring a segment of code pointless - download the code and make the change and RUN the change.
- Feel free to post a link to what you think the code should be.

# You've done this before...

- If you aren't actually downloading and running the code - please don't review my PRs.
- I would never intentionally waste someone's time and send them down a rabbit hole.
- I don't appreciate my time being wasted.